e2e/update_snapshots/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+bazel test //:test
+
+echo "failing snapshot" > __snapshots__/link.test.js.snap
+
+if bazel test //:test; then
+  echo "ERROR: expected snapshot test to fail!"
+  exit 1
+fi
```

Comment:
Does `bazel test //:test` resolve to a truthy value in bash when it fails? Reading this I would think it would return `0` and the condition wouldn't execute.

Reply:
bash if statements are backwards
```
if cmd; then
   // will enter if cmd succeeds (exits 0)
fi
```